core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/GoogleReplacementMode.kt

```diff
@@ -61,5 +69,6 @@ public val GoogleReplacementMode.playBillingClientMode: Int
         GoogleReplacementMode.WITH_TIME_PRORATION -> 1
         GoogleReplacementMode.CHARGE_FULL_PRICE -> 5
         GoogleReplacementMode.CHARGE_PRORATED_PRICE -> 2
+        GoogleReplacementMode.DEFERRED -> 6
```

Comment:
I believe this number should be `4`. It will be `6` once we update to BC7 in the next major (coming soon). Reason is the current version is still using the deprecated `ProrationMode` numbers.

Reply:
good catch!

Reply:
Oh wow, very good catch! Is it still using `ProrationMode` even if we're calling `googleReplacementMode()` in the purchase methods? See e.g. [here](https://github.com/RevenueCat/purchases-kmp/blob/26245aab0fb822bc9226cf3626ea923828540fd6/core/src/androidMain/kotlin/com/revenuecat/purchases/kmp/Purchases.android.kt#L187).

Reply:
Ohhh right, ignore me 🤦‍♂️. I was thinking this was calling the PHC APIs, but we are not so this is ok. See how the current PHC APIs do use proration mode: https://github.com/RevenueCat/purchases-hybrid-common/blob/e46ac1eab1a4c3e2c1f73232c8017b7e972981a8/android/hybridcommon/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt#L605

But yeah, since this has its own implementation, I think this is correct! Sorry for the confusion!

Reply:
No worries, thanks for raising it! Good to be aware of these things!